components/reddit/reddit.app.mjs

```diff
@@ -138,10 +258,10 @@ module.exports = {
           params,
         }));
     },
-    async getNewSubredditLinks(before, subreddit, limit = 100) {
+    async getNewSubredditLinks(subreddit, opts) {
```

Comment:
Since the function parameters for `getNewSubredditLinks` have been changed, I think we'll need to update how `getNewSubredditLinks` is called in `sources/new-comments-on-a-post.mjs` and `sources/new-links-on-a-subreddit.mjs`.